ui/.env.example

```diff
@@ -0,0 +1,8 @@
+# Important: Inline comments are currently not supported in this env file
+
+VITE_API_PASSWORD=password-api
+VITE_API_URL=http://localhost:3002/v1
+VITE_API_USERNAME=user-api
+VITE_ISSUER_DID=
+VITE_ISSUER_LOGO=
+VITE_ISSUER_NAME=
```

Comment:
I would also add the placeholder "Local issuer" or something distinct from the root example.